SPECS/irqbalance/irqbalance.spec

```diff
@@ -56,6 +57,10 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_datadir}/*
 
 %changelog
+* Mon Jul 01 2024 Cameron Baird <cameronbaird@microsoft.com> - 1.9.3-2
```

Comment:
Date itself looks correct, just looking whether this is intentional an older date?